owl/EASE.owl

```diff
@@ -1144,6 +1189,42 @@ The second issue relates to what is &quot;outside&quot;. The state is a configur
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE.owl#3DPosition -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE.owl#3DPosition">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE.owl#EuclideanPoint"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/EASE.owl#hasTranslation"/>
+                <owl:someValuesFrom rdf:resource="http://www.w3.org/2001/XMLSchema#string"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment>A point in three dimensional space, given as translation.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE.owl#6DPose -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE.owl#6DPose">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE.owl#EuclideanPoint"/>
```

Comment:
Euclidean points, and points in general, are often not regarded as having an orientation. This is because usually when one speaks of a point in Euclidean space, they mean a linear space such as R^3, not the special Euclidean group SE(3).

Typically, a point might be regarded as a part of some orientable mathematical object.

Reply:
I was thinking more general, "a point is an element of some space".

Reply:
Well since you did not make a suggestion I will just remove the concept EuclideanPoint